plugins/inputs/sqlserver/README.md

```diff
@@ -46,6 +46,11 @@ GO
   #  "Server=192.168.1.10;Port=1433;User Id=<user>;Password=<pw>;app name=telegraf;log=1;",
   # ]
 
+  ## This enables a specific set of queries depending on the database type. If specified, it replaces azuredb = true/false and query_version = 2
+  ## In the config file, the sql server plugin section should be repeated multiple times each with a set of servers for a specific database_type
+  ## possible values for database_type are  "AzureSQLDB" or  "SQLServer" or "AzureSQLManagedInstance"
```

Comment:
I just think a list is easier to spot and read
```
## possibles values are:
## - "SQLServer"
## - "AzureSQLDB"
## - "AzureSQLManagedInstance"
```

Reply:
Done